include/roaring/roaring.h

```diff
@@ -383,8 +383,8 @@ bool roaring_bitmap_contains_range(const roaring_bitmap_t *r,
                                    uint64_t range_end);
 
 /**
- * Check if an items is present, using context from a previous insert for speed
- * optimization.
+ * Check if an items is present, using context from a previous insert or search
```

Comment:
Here previous doc says "from a previous insert", but I found previous searching is also reuseable, so change it.